examples/dt/uni07eta/control-plane/service-values.yaml

```diff
@@ -0,0 +1,173 @@
+        netapp_storage_family=ontap_cluster
+        consistencygroup_support=True
+      networkAttachments:
+        - storage
+      customServiceConfigSecrets:
+        - cinder-volume-ontap-secrets
+  cinder-volume-nfs-secrets: |
+    [nas]
+    nas_host = _replaced_
+    nas_share_path = _replaced_
+  cinder-volume-ontap-secrets: |
+    [ontap]
+    netapp_login = _replaced_
+    netapp_password = _replaced_
+    netapp_vserver = _replaced_
+
+  swift:
+    enabled: true
+
+  glance:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = primary:cinder,secondary:swift
+      [glance_store]
+      default_backend = primary
+      [primary]
+      rootwrap_config = /etc/glance/rootwrap.conf
+      description = NFS based cinder store
+      cinder_store_auth_address = {{ .KeystoneInternalURL }}
+      cinder_store_user_name = {{ .ServiceUser }}
+      cinder_store_password = {{ .ServicePassword }}
+      cinder_store_project_name = service
+      cinder_catalog_info = volumev3::internalURL
+      cinder_use_multipath = true
+      [secondary]
+      swift_store_create_container_on_put = True
+      swift_store_auth_version = 3
+      swift_store_auth_address = {{ .KeystoneInternalURL }}
+      swift_store_endpoint_type = internalURL
+      swift_store_user = service:glance
+      swift_store_key = {{ .ServicePassword }}
+      [oslo_concurrency]
+      lock_path = /var/lib/glance/tmp
+    databaseInstance: openstack
+    glanceAPIs:
+      default:
+        preserveJobs: false
+        replicas: 1
+        type: split
+
+  ironic:
+    enabled: true
+
+  octavia:
+    enabled: false  # TODO after healthmanager fixed
+    amphoraImageContainerImage: quay.io/gthiemonge/octavia-amphora-image
+    apacheContainerImage: registry.redhat.io/ubi9/httpd-24:latest
+    octaviaAPI:
+      networkAttachments:
+        - internalapi
+    octaviaHousekeeping:
+      networkAttachments:
+        - octavia
+      customServiceConfig: |
+        [controller_worker]
+        loadbalancer_topology=ACTIVE_STANDBY
+    octaviaHealthManager:
+      networkAttachments:
+        - octavia
+      customServiceConfig: |
+        [controller_worker]
+        loadbalancer_topology=ACTIVE_STANDBY
+    octaviaWorker:
+      networkAttachments:
+        - octavia
+      customServiceConfig: |
+        [controller_worker]
+        loadbalancer_topology=ACTIVE_STANDBY
+
+
+  heat:
+    enabled: true
+
+  redis:
+    enabled: true
+
+
+  ovn:
+    ovnController:
+      nicMappings:
+        datacentre: ocpbr
+        octavia: octbr
+      availability-zones:
+        - zone-1
+
+  neutron:
+    customServiceConfig: |
+      [DEFAULT]
+      vlan_transparent = true
+      agent_down_time = 600
+      router_distributed = true
+      router_scheduler_driver = neutron.scheduler.l3_agent_scheduler.ChanceScheduler
+      allow_automatic_l3agent_failover = true
+      debug = true
+      default_availability_zones = zone-1,zone-2
```

Comment:
Likewise, we don't need this line as we are not focused on OVN Availability zones.

Reply:
done